index.html

```diff
@@ -15,6 +15,9 @@ <h2>Load your .json, .tachibk or .proto.gz file</h2>
             <button id="use-demo-data">Use Demo Data</button>
         </div>
     </div>
+    <div class="settings-icon" id="settings-icon">
+        &#9881;
+    </div>
```

Comment:
Should probably replace this with material symbol once #23 gets merged.

Reply:
Reminder that it was merged

Reply:
Yeah now that #23 has been merged I'll look into swapping it soon.